### PR TITLE
Add stub for new exercise: sum-of-multiples

### PR DESCRIPTION
**Pull request by kytrinyx**
This is a proposal for a new exercise based on the canonical data in the problem-specifications repository.

I didn't make any design choices whatsoever—if you think a different API would be better suited to this exercise, please suggest some changes, and I will regenerate the test suite.

I put myself as contributor rather than author, since I don't intend to write the example solution.
As such, the authorship of this exercise is up for grabs to anyone who wants to spend a few minutes writing some code that will pass the tests.

### CODE REVIEW

**Comment by Stargator** [comment]
@kytrinyx The sum-of-multiplesexercise is missing the pubspec.yaml file.

**Comment by kytrinyx** [comment]
@Stargator there's a `pubspec.yaml` file in this PR. Is there a different pubspec that is missing? Or is it in the wrong place?

**Comment by Stargator** [comment]
> there's a pubspec.yaml file in this PR. Is there a different pubspec that is missing? Or is it in the wrong place?

No, it's fine. I guess I just missed it.

File: exercises/practice/sum-of-multiples/test/sum_of_multiples_test.dart [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import 'package:test/test.dart';
```
Comment (Stargator) [inline exercises/practice/sum-of-multiples/test/sum_of_multiples_test.dart:1]: This is missing the import from sum_of_multiples.dart.

**Review by Stargator — CHANGES_REQUESTED** [review]
I am working on the solution for this now

**Comment by Stargator** [comment]
@kytrinyx I've implemented a solution and added the missing import in the test suite. I don't know if you want to regenerate the test suite. If not, then this is good.

**Comment by kytrinyx** [comment]
I don't need to regenerate the test suite (I'll add the import on my end so I have it next time I have to touch the exercise).

I'll add you as author and then get this merged.

Thank you so much for taking the time to implement the solution!

**Review by Stargator — APPROVED** [review]

**Comment by kytrinyx** [comment]
Thanks @Stargator!

**Pull request merged**